servicebroker_node/lib/api/api.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function() {
+
+  var fs = require('fs');
+  var path = require('path');
+
+  var settings = JSON.parse(
```

Comment:
We need to pick one. If it is going to be static, we might has well hardcode it in a file which is read once and always returned. It is a bit of overkill to regenerate the map each time getCatalog is called.